lit/docs/operation/creds/aws-secretsmanager.lit

```diff
@@ -6,12 +6,20 @@
 \section{
   \title{Configuration}
 
-  The ATC is configured with an access key and secret key or session token and
-  the AWS region that your parameters are stored within. If no access key,
-  secret key, or session token is provided, Concourse will attempt to use
-  environment variables or the instance credentials assigned to the instance.
+  In order to integrate with AWS Secrets Manager for credential management, the
+  web node must be configured with:
 
-  The ATC's configuration specifies the following:
+  \list{
+    an access key and secret key, or a session token
+  }{
+    the AWS region that your parameters are stored within.
+  }
+
+  If no access key, secret key, or session token is provided, Concourse will
+  attempt to use environment variables or the instance credentials assigned to
+  the instance.
+
+  The web nodes's configuration specifies the following:
```

Comment:
Just a typo, I _think_ the possessive form of the plural noun would be: web nodes' configuration.